src/hooks/useAccounts.tsx

```diff
@@ -224,6 +230,8 @@ const useAccountsContext = () => {
 
           if (walletConnectionType === WalletConnectionType.Privy && authenticated && ready) {
             try {
+              // Give Privy a second to finish the auth flow before getting the signature
+              await sleep();
```

Comment:
Not sure I fully understand this. Is this because of the inclusion of the MFA modal?

Reply:
Privy `login()` was failing with `exited_auth_flow` because you are fetching the signature as soon as `privy.ready` and `privy.authenticated` are set. This was causing that the `onComplete` callback is not executed. We found out that we need a small amount of time for the auth cycle to end properly before you can fetch the signature, so I added a sleep call before you do.

Reply:
Got it. Not too familiar but I wonder if there is a way to check for something other than ready and authenticate to mitigate this.